### PR TITLE
Register sushi.is-a.dev

### PR DESCRIPTION
**Pull request by sushi-ae**
Added `sushi.is-a.dev` using the [CLI](https://www.npmjs.com/package/@is-a-dev/cli).

### CODE REVIEW

File: domains/sushi.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+    "owner": {
+        "username": "sushi-ae",
+        "email": "frapdotpng@gmail.com"
+    },
+
+    "record": {
+        "CNAME": "sushi-ae.github.io"
```
Comment (DEV-DIBSTER) [inline domains/sushi.json:8]: Just redirects to our site. Please have an actual site.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

File: domains/sushi.json (hunk 1 above)
Comment (sushi-ae) [reply domains/sushi.json:8]: why does it do that, I didn't even set an alternate domain yet

Comment (sushi-ae) [reply domains/sushi.json:8]: even if I unpublish the site, sushi-ae.github.io still redirects to is-a.dev

Comment (sushi-ae) [reply domains/sushi.json:8]: any ideas why this happening? or how to potentially fix it?

**Pull request closed without merging**